src/main/drivers/pwm_mapping.h

```diff
@@ -59,15 +59,12 @@ typedef struct drv_pwm_config_s {
 #ifdef STM32F303xC
     bool useUART3;
 #endif
-#if defined(STM32F40_41xxx) || defined (STM32F411xE)
+#ifdef STM32F4
     bool useUART2;
     bool useUART6;
 #endif
     bool useVbat;
 	bool useOneshot;
```

Comment:
I've only left it in given it is in the configurator - we could kill it - and just go with the pwm_rate==0